tripy/examples/nanogpt/model.py

```diff
@@ -108,9 +103,7 @@ def __call__(self, x: tp.Tensor):
         # (batch_size, num_heads, seq_len, seq_len) @ (batch_size, num_heads, seq_len, head_size) -> (batch_size, num_heads, seq_len, head_size)
         out = att @ v
         out = tp.cast(out, x.dtype)
-        out_shape = tp.concatenate([B, T, tp.Tensor([self.embedding_size])], dim=0)
-        # WAR to prevent computing output rank in infer_rank for reshape, will be addressed by #228
-        out_shape.trace_tensor.shape = (3,)
+        out_shape = [B, T, self.embedding_size]
         out = tp.reshape(tp.transpose(out, 1, 2), out_shape)
```

Comment:
We can probably just inline the shapes here and above

Reply:
Updated.